test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -242,6 +248,21 @@ generic_end_per_testcase(CaseName, Config) ->
 %% Service discovery test
 %%--------------------------------------------------------------------
 
+test_host_refreshing(_Config) ->
+    eventually_true("wait for refresher to converge trees states",
+                    fun() -> trees_for_connections_present() end, ?HOSTS_REFRESH_INTERVAL, 2),
+    ConnectionSups = out_connection_sups(asia_node),
+    [EuropeHost] = lists:filtermap(fun({europe_node1, Host, _}) -> {true, list_to_binary(Host)};
+                                      (_) -> false end, get_hosts()),
+    EuropeSup = rpc(asia_node, mod_global_distrib_utils, server_to_sup_name, [EuropeHost]),
+    [EuropePid] = lists:filtermap(fun({Sup, Pid, supervisor, _}) -> {Sup =:= EuropeSup, Pid};
+                                     (_) -> false end,
+                                      ConnectionSups),
+    erlang:exit(EuropePid, kill),
```

Comment:
I think a comment would be useful here, that it's OK to `kill` the process as it has `temporary` restart type.